ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -778,17 +862,18 @@ impl<Register: Send + Sync + 'static> TraceBuilder<Register> {
         let is_recursive = self.frames.iter().any(|f| f.funcidx == Some(*callee));
 
         let func = self.aot_mod.func(*callee);
-        if inst.is_mappable_call(self.aot_mod)
+        let mappable = !func.is_declaration();
```

Comment:
This variable feels a bit weird and isn't used anywhere except the next line? Maybe just inline it into the `if` so it's `if !func.is_declaration()`?